cmd/embedded-cluster/version.go

```diff
@@ -107,7 +107,7 @@ var metadataCommand = &cli.Command{
 		if err != nil {
 			return fmt.Errorf("unable to marshal versions: %w", err)
 		}
-		fmt.Println(string(data))
+		fmt.Print(string(data))
```

Comment:
was there an extra trailing newline here?

Reply:
Gosh. Restored the original form.